src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Passive scanner rules (alpha)</name>
-	<version>10</version>
+	<version>11</version>
```

Comment:
The version should be kept at 10 (should be bumped only after a new release to marketplace).